codespell_lib/data/dictionary.txt

```diff
@@ -45270,18 +45339,31 @@ valuling->valuing
 valus->values, value, talus,
 valuse->values, value,
 valye->value, valse, valve,
+valyes->values, valses, valves,
 vanishs->vanishes
 varable->variable
 varables->variables
 varagated->variegated
 varaiable->variable
 varaiables->variables
 varaiance->variance
+varaiances->variances
+varaiant->variant
+varaiants->variants
 varaiation->variation
+varaiations->variations
 varaible->variable
 varaibles->variables
+varaince->variance
+varainces->variances
 varaint->variant
 varaints->variants
+varaition->variation
+varaitions->variations
+varance->variance
```

Comment:
Since **b** and **v** are close on many keyboards, perhaps:
```
varance->variance, balance,
```

Reply:
`varance->balance` requires 2 typos. Also, I couldn't find this typo.